sass/components/_download.scss

```diff
@@ -8,6 +8,7 @@
 .sc-download {
     padding: 24px;
     display: flex;
+    border-radius: 4px;
```

Comment:
Could we have this as a variable like we have it in **deriv-app**? (`$border_radius`)

Reply:
Right, let's use it